### DISCUSSION TITLE
unable to create a bucket

### DISCUSSION THREAD

**Discussion by HassanIyan** [supabase/supabase#4431] [Questions]
when i try to create a bucket from supabase dashboard it shows a pop up saying

`column "public" of relation "buckets" does not exist`

how do we fix this

**Comment by patrickmonteiro** [comment]
I'm having the same error! I can't create a bucket, but I've noticed that this message only appears on projects a little older, I have another newer project and apparently the problem doesn't happen.


<img width="949" alt="Captura de Tela 2022-01-03 às 10 15 54" src="https://user-images.githubusercontent.com/13258255/147934714-5f5ddf70-63fc-4067-862b-8d5d8bdeb836.png">

<img width="955" alt="Captura de Tela 2022-01-03 às 10 16 22" src="https://user-images.githubusercontent.com/13258255/147934778-598497d9-b6e1-4d15-acc8-945708be78ed.png">

**Comment by GaryAustin1** [comment]
Sounds like for some reason your storage-api on the server did not get migrated/updated.

https://github.com/supabase/storage-api/blob/master/migrations/tenant/0007-add-public-to-buckets.sql
ALTER TABLE storage.buckets ADD COLUMN "public" boolean default false;

About 5 months ago it was added.

**Reply by venkateshwarv** [reply]
I ran this query SQL Editor from UI. It then started to work. Thanks!!!
